bionic/deriver.py

```diff
@@ -463,6 +468,9 @@ class Bootstrap:
     executor = attr.ib()
     should_memoize_default = attr.ib()
 
+    def evolve(self, executor):
```

Comment:
Rather than having a method specifically for `executor`, should we just accept `**kwargs` here?